### PR TITLE
BeLL Apps tutorial (fixes issue #1225)

### PR DESCRIPTION
**Pull request by jwill12**
https://rawgit.com/jwill12/jwill12.github.io/iss3/#!pages/bellapps.md

### CODE REVIEW

File: pages/bellapps.md [hunk 1]
```diff
@@ -34,22 +34,24 @@ Fill out the information, although as we are just doing this as a test, accuracy
...
-There are three other important kinds of updates that you receive on the community side: updates, publications, and surveys. 
+There are three other important kinds of updates that you receive on the community side: updates, publications, and surveys.
 
 As you can see from the image below, there is an update ready to be downloaded. Usually, next to the update, you should also see  publication(s) ready to be downloaded.
 
 ![Update from the nation](uploads/images/update_publication.jpg "Dashboard in your localhost")
 
 First, click the "Update Available" button and it will reload your homepage with a successful update message. An update refers to a new software update which improves the software. Next, click on "Publications", under the Manager page, and sync the publications. Publications add new resources or courses to your library. Last, repeat the process of sending an activities sync to the nation.
 
+**NOTE**: If `Update Available` or `Publications`, is not visible, continue on with tutorial. Will only display if necessary!
```
Comment (kylemathias) [inline pages/bellapps.md]: add an "They" to "Will only display if necessary!". Should show as "They will only display if necessary!"

File: pages/bellapps.md [hunk 2]
```diff
@@ -34,22 +34,24 @@ Fill out the information, although as we are just doing this as a test, accuracy
...
-There are three other important kinds of updates that you receive on the community side: updates, publications, and surveys. 
+There are three other important kinds of updates that you receive on the community side: updates, publications, and surveys.
 
 As you can see from the image below, there is an update ready to be downloaded. Usually, next to the update, you should also see  publication(s) ready to be downloaded.
 
 ![Update from the nation](uploads/images/update_publication.jpg "Dashboard in your localhost")
 
 First, click the "Update Available" button and it will reload your homepage with a successful update message. An update refers to a new software update which improves the software. Next, click on "Publications", under the Manager page, and sync the publications. Publications add new resources or courses to your library. Last, repeat the process of sending an activities sync to the nation.
 
+**NOTE**: If `Update Available` or `Publications`, is not visible, continue on with tutorial. They will only display if necessary!
```
Comment (xyb994) [inline pages/bellapps.md]: Is the `,` after `Publications` necessary? Would it be more friendly to use `.` instead of `!` at the end of the sentence?

**Comment by kylemathias** [comment]
@jwill12 fix the conflicts and we can merge. let me know in Gitter once you have done so.

**Comment by xyb994** [comment]
Fixed in PR #1637. Close #1225.

**Pull request closed without merging**